src/app.ts

```diff
@@ -1,8 +1,130 @@
 import express from 'express'
+import { celebrate, Joi, isCelebrateError, Segments } from 'celebrate'
+import { getClient } from './databaseConnection'
+// import getAuth from 'firebase-auth'
```

Comment:
I think we shouldn't keep code that isn't working in comments. I belive it should be deleted.